service-fabrik-plugin.go

```diff
@@ -146,18 +146,66 @@ func (serviceFabrikPlugin *ServiceFabrikPlugin) Run(cliConnection plugin.CliConn
 			//Internally split into start and abort.
 			switch cmds[0] {
 			case "start":
-				if argLength != 3 {
-					errors.IncorrectNumberOfArguments()
-				}
-				fmt.Println("Are you sure you want to start restore? (y/n)")
-				var userChoice string
-				fmt.Scanln(&userChoice)
-				if userChoice == "y" {
-					restore.NewRestoreCommand(cliConnection).StartRestore(cliConnection, args[1], args[2])
+				if argLength == 4 {
+					if args[2] == "--backup_guid" {
+						fmt.Println("Are you sure you want to start restore? (y/n)")
+						var userChoice string
+						fmt.Scanln(&userChoice)
+						if userChoice == "y" {
+							restore.NewRestoreCommand(cliConnection).StartRestore(cliConnection, args[1], args[3], "", true, false, false, "", "", false)
+						} else {
+							os.Exit(7)
+						}
+					} else if args[2] == "--timestamp" {
+						fmt.Println("Are you sure you want to start restore? (y/n)")
+						var userChoice string
+						fmt.Scanln(&userChoice)
+						if userChoice == "y" {
+							restore.NewRestoreCommand(cliConnection).StartRestore(cliConnection, args[1], "", args[3], false, false, false, "", "", false)
+						} else {
+							os.Exit(7)
+						}
+					} else {
+						errors.InvalidArgument()
+					}
+				} else if argLength == 6 {
+					if args[4] == "--from-instance-id" {
+						fmt.Println("Are you sure you want to start restore? (y/n)")
+						var userChoice string
+						fmt.Scanln(&userChoice)
+						if userChoice == "y" {
+							restore.NewRestoreCommand(cliConnection).StartRestore(cliConnection, args[1], "", args[3], false, true, false, args[5], "", false)
+						} else {
+							os.Exit(7)
+						}
+					} else if args[4] == "--from-instance" {
+						fmt.Println("Are you sure you want to start restore? (y/n)")
+						var userChoice string
+						fmt.Scanln(&userChoice)
+						if userChoice == "y" {
+							restore.NewRestoreCommand(cliConnection).StartRestore(cliConnection, args[1], "", args[3], false, false, true, "", args[5], false)
+						} else {
+							os.Exit(7)
+						}
+					} else {
+						errors.InvalidArgument()
```

Comment:
can we provide a sample help command instead of throwing only InvalidArgument() ?